lib/tasks/try-each.js

```diff
@@ -19,18 +19,11 @@ module.exports = CoreObject.extend({
                                                 dependencyManagerAdapters: dependencyManagerAdapters
     });
 
-    var shutdownCount = 0;
+    task._canceling = false;
     task._on('SIGINT', function() {
-      if (shutdownCount === 0) {
```

Comment:
FWIW, this two phase exit system will not work on recent ember-cli anyways because `process.exit()` no longer ensures immediate exit (as it did before).